### ISSUE TITLE
Video input from webcam not working

### ISSUE THREAD

**Issue by sandstorm12** [gradio-app/gradio#780]
### Describe the bug

I have tested this feature using your demo available on [Colab](https://colab.research.google.com/drive/1hWvmAfh_2TVxK5D0_EsJuGYTDqMQTjrx). Also, have tried the same code on local ubuntu (20.04) machine. I have opened the web UI on the latest versions of chrome and firefox on ubuntu and the latest chrome and firefox version on android. In all scenarios, the Gradio UI is just a frozen screen doing nothing.

### Reproduction

Try to run the video input demo in google colab [here](https://colab.research.google.com/drive/1hWvmAfh_2TVxK5D0_EsJuGYTDqMQTjrx)

### Screenshot

![Screenshot from 2022-03-05 11-53-36](https://user-images.githubusercontent.com/12384476/156875185-6951d1c9-f807-4097-adb7-9eb78205febc.png)

### Logs

```shell
There is no particular error or warning
```

### System Info

```shell
Gradio 2.8.7
Ubuntu 20.04
Chrome 98
Firefox 97

Used my working laptop (HP pavilion) and Android 9 camera.
```

### Severity

blocker

**Comment by aryan461** [comment]
Same for me. Even the browser doesn't ask for access to my webcam. Simply nothing happens.

**Comment by abidlabs** [comment]
Ah thank you for bringing this to our attention @sandstorm12, @aryan461. We are looking into this.  In the meantime, please use an older version of Gradio. `pip install gradio==2.7.5` seems to work

**Comment by abidlabs** [comment]
@dawoodkhan82 is this something you can look into?

**Comment by aliabid94** [comment]
Fixed in 2.8.8. Let us know if you have any other issues!
